easybuild/easyconfigs/r/RStan/RStan-2.21.5-foss-2021b-R-4.1.2.eb

```
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
+
+name = 'RStan'
+version = '2.21.5'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://github.com/stan-dev/rstan'
+description = """RStan is the R interface to Stan. Stan is a state-of-the-art
+ platform for statistical modeling and high-performance statistical computation."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('R', '4.1.2')
+]
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib',
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz',
+}
+
+exts_list = [
+    ('inline', '0.3.19', {
+        'checksums': ['0ee9309bb7dab0b97761ddd18381aa12bd7d54678ccd7bec00784e831f4c99d5'],
+    }),
+    ('loo', '2.5.1', {
+        'checksums': ['866a2f54a4e8726cc3062e27daa8a073e6ac4aeb6719af7845284f7a668745f1'],
+    }),
+    ('StanHeaders', '2.21.0-7', {
+        'installopts': '--configure-args="CXXFLAGS=\"$CXXFLAGS -DSTAN_THREADS\""',
```

This has the same issue.

```suggestion
        'installopts': '--configure-args="CXXFLAGS=$CXXFLAGS -DSTAN_THREADS"',
```